CentralManagement/ChocoCCM Command Reference/SetCCMNotificationStatus.md

```diff
@@ -0,0 +1,87 @@
+# Set-CCMNotificationStatus
+
+<!-- This documentation is automatically generated from /Set-CCMNotificationStatus.ps1 using GenerateDocs.ps1. Contributions are welcome at the original location(s). -->
+
+Turn notifications on or off in CCM
+
+## Syntax
+
+~~~powershell
+Set-CCMNotificationStatus `
+  -Enable [<CommonParameters>]
+~~~
+
+
+~~~powershell
+Set-CCMNotificationStatus `
+  -Disable [<CommonParameters>]
+~~~
```

Comment:
This should be a pair of `Enable`/`Disable` commands rather than two switches on one command, unless there's a lot more that would be on this command?

Reply:
At the moment, it's the one thing. But, I dunno what the plan was for the notification section, if there _could_ be more etc. Hence the design decision here.